### PR TITLE
fix: prints removed and logo added

### PR DESCRIPTION
**Pull request by tomaszandel**
- pritns changed to logs
- logo support added

### CODE REVIEW

**Review by maria-j-k — APPROVED** [review]

File: backend/app/routes/web/search_results.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ async def search_post(
                 )
                 res_json["response"]["docs"] = copy.deepcopy(new_docs)
             except (Exception,):  # pylint: disable=broad-except
-                print("Something goes wrong..")
+                logger.exception("Exception happened during related services extension")
```
Comment (Michal-Kolomanski) [inline backend/app/routes/web/search_results.py:97]: `except Exception as e `
Return `e` in the log

Not the biggest fan of such a broad catch. If possible except only concrete exceptions.

File: ui/apps/ui/src/app/components/results-with-pagination/result.component.html [hunk 2]
```diff
@@ -1,6 +1,33 @@
-<div class="type-icon-holder {{ type.label }}">
-  <div class="type-icon"></div>
-</div>
+<ng-container
+  *ngIf="
+    type.value === 'service' ||
+      type.value === 'data-source' ||
+      type.value === 'provider' ||
+      type.value === 'data source';
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/results-with-pagination/result.component.html:6]: Why `data-source` and `data source`?

Comment (Michal-Kolomanski) [reply ui/apps/ui/src/app/components/results-with-pagination/result.component.html:6]: By the way, offers in bundles also should have icons, right? @agpul

Comment (tomaszandel) [reply ui/apps/ui/src/app/components/results-with-pagination/result.component.html:6]: old code, its just a reformat :)

**Pull request merged**